pages/solutions/[id].tsx

```diff
@@ -82,18 +80,12 @@ export default function Comparison({ pageProps }) {
         </Container>
       </div>
       <FeatureBanner features={comp.features} />
+      <Container className="col-9 col-md-4"></Container>
```

Comment:
I think this empty container is doing nothing?

Reply:
Ah, thanks.  Stray container from trying to add in-page navigation.

Reply:
> Notes:
> 
> * The Dropdown has a good layout on mobile too
> * Aesthetically, perhaps the headers in the dropdown need either increased padding/boldness.  On mobile (attached), it seems like they are very close to the same font as the rest of the menu (Attached)
> 
> Bugs:
> 
> * Solutions/redshift doesn't go to the right page
> * Looks like there are some git troubles here.  This PR has commits about adding comparison pages too.
> 
> ![IMG_785434C4AE74-1](https://user-images.githubusercontent.com/303226/116938257-51b96400-ac1f-11eb-9651-898965231ac8.jpeg)

Yes, I branched off of a branch so the commit history is jumbled.  What would be the right way to fix that without losing changes?  A rebase?